website/docs/guides/docs/sidebar/autogenerated.md

```diff
@@ -194,6 +194,98 @@ Naming your introductory document `README.md` makes it show up when browsing the
 
 :::
 
+### Customizing category index matching
+
+It is possible to opt out any of the category index conventions, or define even more conventions. You can inject your own `isCategoryIndex` matcher through the [`sidebarItemsGenerator`](#customize-the-sidebar-items generator) callback. For example, you can also pick `intro` as another file name eligible for automatically becoming the category index.
```

Comment:
Link is failing in preview:

![image](https://user-images.githubusercontent.com/749374/151174597-3c53ea9f-f7c0-4007-b965-e4dab45caae5.png)

https://deploy-preview-6451--docusaurus-2.netlify.app/docs/sidebar/autogenerated/#customizing-category-index-matching